cli/ops/compiler.rs

```diff
@@ -1,152 +1,13 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
-use super::dispatch_json::Deserialize;
-use super::dispatch_json::JsonOp;
-use super::dispatch_json::Value;
-use crate::futures::future::try_join_all;
-use crate::op_error::OpError;
 use crate::state::State;
 use deno_core::CoreIsolate;
-use deno_core::ModuleLoader;
-use deno_core::ModuleSpecifier;
-use deno_core::ZeroCopyBuf;
-use futures::future::FutureExt;
 
-pub fn init(i: &mut CoreIsolate, s: &State) {
-  i.register_op("op_resolve_modules", s.stateful_json_op(op_resolve_modules));
-  i.register_op(
-    "op_fetch_source_files",
-    s.stateful_json_op(op_fetch_source_files),
-  );
-  let custom_assets = std::collections::HashMap::new(); // TODO(ry) use None.
+pub fn init(i: &mut CoreIsolate, _s: &State) {
+  let custom_assets = std::collections::HashMap::new();
+  // TODO(ry) use None.
+  // TODO(bartlomieju): is this op even required?
```

Comment:
It could not be required... but it means we need to catch these dependencies when using `lib` with the runtime compiler APIs.

Reply:
Not sure I understand, I think if we loaded all available libs during snapshot it could be removed

Reply:
oh, yeah I forgot we did that...